README.md

```diff
@@ -1,4 +1,6 @@
 # time-tracker
 Java (Maven) application for tracking time on the job
 
-Time tracker
+##  Purpose
```

Comment:
This is just added for clarification. and to see if comments on codes really shows on pull req page.